### ISSUE TITLE
Update BLS12-381 library version

### ISSUE THREAD

**Issue by marsella** [boltlabs-inc/libzkchannels-crypto#73]
This dependency got an update to version 0.5.0, but it breaks some stuff.

**Comment by indomitableSwan** [comment]
@yaymukund @gijsvl Am I correct in understanding that the dependency conflict should be fixed by the end of the year, which will allow us to use the latest version?

**Comment by yaymukund** [comment]
> @yaymukund @gijsvl Am I correct in understanding that the dependency conflict should be fixed by the end of the year, which will allow us to use the latest version?

Yes, specifically https://github.com/myrrlyn/funty/issues/3#issuecomment-981048749 looks like it should fix it.
